.github/workflows/ci.yml

```diff
@@ -1,73 +1,171 @@
 name: CI
```

Comment:
this file is generated from ember-ci-update and the ci.yml at the root of the monorepo